unstructured/partition/pdf.py

```diff
@@ -1002,10 +1105,9 @@ def get_word_bounding_box_from_element(
 
                 # TODO(klaijan) - isalnum() only works with A-Z, a-z and 0-9
                 # will need to switch to some pattern matching once we support more languages
-                if index == 0:
+                if not word:
                     isalnum = char.isalnum()
-
-                if char.isalnum() != isalnum:
+                if word and char.isalnum() != isalnum:
```

Comment:
if i'm following this logic correctly, can this be simplified to:

    isalnum = char.isalnum()
    if word and isalnum == False:
        ....

Reply:
This condition here is to use `isalnum` flag to help us split the `word` by alphanumeric and non-alphanumeric property in the text sentence. For example, `Lorem ipsum dolor sit amet, consectetur "adipiscing" elit.` would be split as 

```
Lorem 
ipsum 
dolor 
sit 
amet
, 
consectetur
"
adipiscing
" 
elit
.
```

Reply:
@Klaijan Did you got a test PDF to test the function? if not https://documentation.nokia.com/sr/23-7-2/pdf/Interface_Configuration_Guide_23.7.R1.pdf